src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/AnswerSteps.java

```diff
@@ -77,7 +98,8 @@ public void an_existing_question_with_id() throws IOException {
         response = restTemplate.exchange(baseUrl + endpoints.get("question"), HttpMethod.POST, request, String.class);
         String json = response.getBody();
         question = (Question) JsonUtils.toObjectFromJson(json, Question.class);
-        this.questionId = question.getQuestionId();
+        this.currentQuestionId = question.getQuestionId();
+        questionIds.add(question.getQuestionId());
     }
 
     @Given("^a standard answer$")
```

Comment:
reason for creating an instance of the jsonutils class when it only has static methods?

Reply:
It's a utility class and re-usable from anywhere. There's no need to classes to create their own instance of JsonUtils.